README.md

```diff
@@ -1,4 +1,4 @@
-# Astrobee Robot Software
+# Astrobee Robot Software v1
```

Comment:
dude you need to fix that variable

Reply:
nice work you fixed it, don't forget about the importance of global variables next time